test/unit/cog_api/http/groups_test.exs

```diff
@@ -44,4 +55,37 @@ defmodule CogApi.HTTP.GroupsTest do
       end
     end
   end
+
+  describe "group_add_user" do
+    it "adds the user to the group" do
+      cassette "groups_add" do
+        endpoint = valid_endpoint
+        group = Client.group_create(endpoint, %{name: "groups_add"}) |> get_value
+        user = Client.user_create(endpoint, user_params("group_add_user")) |> get_value
+        assert group.users == []
+
+        group = Client.group_add_user(endpoint, group, user) |> get_value
+
+        assert group.users == [user]
+      end
+    end
+  end
+
+  defp user_params(test_name) do
```

Comment:
This was why I was thinking of getting `ex_machina` in here. What are your thoughts on that now?